### PR TITLE
🌐 Add Spanish traslation for Body - Multiple Parameters

### PR DESCRIPTION
**Pull request by anthonycepeda**
Hi Guys :)

According to Spanish Traslation Feature: #1218 this PR includes the spanish traslation for tutorial/body-multiple-params.md

Thanks

### CODE REVIEW

File: docs/es/docs/tutorial/body-multiple-params.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    },
+    "importance": 5
+}
+```
+
+Nuevamente, los tipos de datos serán convertidos, validados, documentados, etc.
+
+## Múltiples parámetros en el cuerpo y consulta
+
+Podrías también declarar parámetros adicionales de consulta en cualquier parte que lo necesites, además de los parámetros en el cuerpo.
```
Comment (bezaca) [inline docs/es/docs/tutorial/body-multiple-params.md:99]:
```suggestion
Puedes declarar parámetros adicionales de consulta en cualquier parte que lo necesites, además de los parámetros en el cuerpo.
```

**Review by bezaca — APPROVED** [review]
Nice job!

**Comment by anthonycepeda** [comment]
thanks @bezaca  for your insights

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**